packages/material-ui-system/src/index.spec.tsx

```diff
@@ -27,19 +27,19 @@ function composeTest() {
   styler({ color: 'test', spacing: 1 });
 }
 
-function cssTest() {
+function sxTest() {
   function styleFunction(props: { color?: string; spacing?: number; theme?: object }) {
     return {};
   }
 
-  const wideOrNarrowStyleFunction = css(styleFunction);
+  const wideOrNarrowStyleFunction = styleFunctionSx(styleFunction);
 
   // narrow
-  wideOrNarrowStyleFunction({ theme: {}, css: { color: 'blue', spacing: 2 } });
-  // wide, undesire: `css` is required, marking it as optional breaks system/basics/#css-property
-  wideOrNarrowStyleFunction({ theme: {}, color: 'blue', spacing: 2, css: {} });
+  wideOrNarrowStyleFunction({ theme: {}, sx: { color: 'blue', spacing: 2 } });
+  // wide, undesire: `sx` is required, marking it as optional breaks system/basics/#css-property
```

Comment:
Reference to deleted section system/basics/#css-property

What does "undesire" mean here?

Reply:
Let me review again the spec and see what is still relevant

Reply:
All css occurences are replaced with `sx` and the `css-property` example is replaced with one for the `sx-property`.

Reply:
What does "undesire" mean here?